iohub/display_utils.py

```diff
@@ -0,0 +1,149 @@
+# Utility functions for displaying data
+
+from typing import Tuple
+
+import numpy as np
+
+from iohub.ngff_meta import ChannelMeta, WindowDict
+
+
+def rgb_to_hex(r: int, g: int, b: int) -> str:
```

Comment:
iohub stores color metadata as a pydantic color field which converts different color representations into hex:
https://github.com/czbiohub/iohub/blob/866e6690991fee623a00b8c60a3602a4d7b30922/iohub/ngff_meta.py#L247

So this function is likely redundant.